ext/mysql2/mysql2_ext.h

```diff
@@ -9,7 +9,13 @@ void Init_mysql2(void);
 #define RSTRING_NOT_MODIFIED
 #include <ruby.h>
 
-#ifdef HAVE_MYSQL_H
+#ifdef HAVE_SERVER_MYSQL_H
+#include <mysql.h>
+#include <server/mysql_com.h>
+#include <errmsg.h>
+#include <mysqld_error.h>
+#include <server/mysql_version.h>
```

Comment:
I wonder why these headers are spread out across two paths like this. What did they change?